docs/podlet/local_development.md

```diff
@@ -1,34 +1,34 @@
 ---
 id: local_development
-title: Local Development
+title: Local development
 ---
 
 It is intended that podlets be developed in isolation from layouts and other podlets. This isolation can introduce challenges into local development due to some components normally provided by a layout (such as headers and Assets) not being available.
 
-## A basic podlet development setup
+## Podlet development setup
 
-At its most basic, the experience of developing a podlet on its own can be as simple as starting the podlet and visiting its URL in your favourite browser.
+The experience of developing a podlet on its own can be as simple as starting the podlet and visiting its URL in your favourite browser.
 
-Consider the following very simple podlet server:
+Consider the following podlet server:
```

Comment:
I really like this removal of "very simple" and things like that. if we want to let the reader know something isn't "how you would do it in production", we should say that 👍